### PR TITLE
[release/0.1] Add missing commits

### PR DESCRIPTION
**Pull request by dgn**

### CODE REVIEW

**Comment by dgn** [comment]
Maybe https://github.com/istio/test-infra/pull/5464 will fix the gen issue

**Comment by dgn** [comment]
fix is in https://github.com/istio-ecosystem/sail-operator/pull/269

File: .devcontainer/devcontainer.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "istio build-tools",
-  "image": "gcr.io/istio-testing/build-tools:master-89ff97db972e1ec7d61a7831bf9408f509e17d73",
+  "image": "gcr.io/istio-testing/build-tools:master-8cdcac292440a2a4c5b6ebaba49bcea2d57d4ecd",
```
Comment (jwendell) [inline .devcontainer/devcontainer.json:3]: should this be 1.22?

**Review by jwendell — APPROVED** [review]

File: .devcontainer/devcontainer.json (hunk 1 above)
Comment (dgn) [reply .devcontainer/devcontainer.json:3]: I'll update to 1.23 once it's released

**Pull request merged**